src/main/resources/swagger/ga4gh-tool-discovery.yaml

```diff
@@ -200,15 +200,9 @@ definitions:
       id:
         type: string
         description: A unique identifier of the tool, scoped to this registry, for example `123456` or `123456_v1`
-      registry:
```

Comment:
Note to self: this deletion makes sense because the registry is duplicated in the image attribute of the ToolVersion anyway.